base/push_deserializer_test.cpp

```diff
@@ -149,6 +189,75 @@ TEST_F(PushDeserializerTest, Stream) {
   EXPECT_EQ(23, stream_.ByteCount());
 }
 
+TEST_F(PushDeserializerTest, DeserializationGipfeli) {
+  Compressor* compressor = google::compression::NewGipfeliCompressor();
```

Comment:
"must be deleted after use", says the comment. We should have a MakeGipfeliCompressor wrapper that returns std::unique_ptr (probably same comment for the decompression test).

Reply:
Done.